packages/api/src/models/medical/patient-import-mapping.ts

```
@@ -0,0 +1,48 @@
+> {
+  static NAME = "patient_import_mapping";
+  declare cxId: string;
+  declare jobId: string;
+  declare rowNumber: number;
+  declare patientId: string;
+  declare requestId: string;
+  declare createdAt: Date;
+  declare updatedAt: Date;
+
+  static initialize(sequelize: Sequelize): void {
+    this.init(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)